hotel-management/src/components/header/Header.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+
+import { Link } from 'react-router-dom';
+
+import Avatar from '@/assets/images/avatar.svg?react';
+import ToggleMenuIcon from '@/assets/images/toggle-menu.svg?react';
+import { ROUTES } from '@/constants/routes';
+import { LinkProps } from '@/types';
+
+import { Button } from '../Button';
+import './index.css';
+
+const Header = () => {
+  const [isDropdownActive, setIsDropdownActive] = useState(false);
+  const { LOGIN_PAGE, REGISTER_PAGE } = ROUTES;
+
+  const handleToggleBtnFilter = () => {
+    setIsDropdownActive(!isDropdownActive);
+  };
+
+  const navbarItems: LinkProps[] = [
+    { textContent: 'Find a Property', href: '#' },
+    { textContent: 'Share Stories', href: '#' },
+    { textContent: 'Rental Guides', href: '#' },
+    { textContent: 'Download Mobile App', href: '#' },
+  ];
+  const btnGroupItems: LinkProps[] = [
+    { textContent: 'Login', href: LOGIN_PAGE },
+    { textContent: 'Register', href: REGISTER_PAGE },
+  ];
+
+  return (
+    <header className="main-header-container">
+      <h1 className="section-title header-title">
+        <Link to="#">ITEL</Link>
+      </h1>
+      <ul className="header-navbar">
+        {navbarItems.map(item => (
+          <li className="text-truncate" key={item.textContent}>
+            <Link to={item.href}>{item.textContent}</Link>
+          </li>
+        ))}
+      </ul>
+      <div className="header-button-group">
+        <Link className="btn btn-primary btn-link" to="#">
+          Become A Host
+        </Link>
+        <div className="wrapper btn-filter-wrapper">
+          <Button onClick={handleToggleBtnFilter} className="btn btn-filter">
```

Comment:
When we define a component, we want to make sure to pass at least prop as possible